css/main.css

```diff
@@ -325,3 +325,66 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+.form-container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 4px 24px;
+}
+
+.form-container h1 {
+  width: 327px;
+  height: 363px;
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+  margin: 24px auto;
+}
+
+.form-group {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  margin: 38px auto;
+}
+
+.form-control input {
+  width: 327px;
+  padding: 15px;
+  background: #fff;
+  border: 1px solid #d0d9d4;
+  box-sizing: border-box;
+  border-radius: 4px;
+  margin: 12px auto;
+}
+
+.form-control textarea {
+  width: 327px;
+  height: 114px;
+  background: #fff;
+  box-sizing: border-box;
+  border-radius: 4px;
+  padding: 12px;
+  margin: 12px auto;
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 24px;
+  color: #091e42;
+  resize: none;
+}
+
+.form-button {
+  color: #fff;
+  background-color: #ff6b00;
+  padding: 15px;
+  width: 128px;
+  height: 48px;
+}
```

Comment:
Kindly add button interactions such as hover and are styled as shown in the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Currently, your button doesn't interact when I hover on it.

![image](https://user-images.githubusercontent.com/38088818/152541362-536d8564-ae68-4fe5-87dd-ae3184ff3ce9.png)